bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/DeviceStatusUpdate.java

```diff
@@ -16,37 +16,41 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * Represents a device status update as represented by the Smart Home Controller.
+ * Represents a device status update as represented by the Smart Home
+ * Controller.
  *
  * @author Stefan Kästle
  *
  */
 public class DeviceStatusUpdate {
 
     /**
-     * {"result":[
-     * ..{
+     * {"result":[ ..{
```

Comment:
I think we should just remove those now :-)

Reply:
Will do :)